tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -398,7 +405,7 @@ private void createFileReport(String name) {
     //            LongStream longStream = tools.parallelStream().filter(Tool::getVerified)
     //                    .mapToLong(tool -> tool.getVersions().parallelStream().filter(ToolVersion::getVerified).count());
     //            System.out.println("Number of versions of tools to test on Dockstore (currently): " + longStream.sum());
-    //            //            toolTestResult = tools.parallelStream().filter(Tool::getVerified).map(this::testTool).reduce(true, Boolean::logicalAnd);
+    //            //            toolTestResult = tools.parallelStream().filter(Tool::getVerified).map(this::testDockstoreTool).reduce(true, Boolean::logicalAnd);
```

Comment:
this part seems odd, a changing section of commented code?

Reply:
Refactoring also changed the comment

Reply:
Just delete it unless there's a good chance that it will used later.
Otherwise, wrap it with a dummy if statement or similar.